packages/review/test/harness/sweep.sh

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Multi-model sweep across all canary fixtures.
+# Usage: OPENROUTER_API_KEY=… ./sweep.sh [model1 model2 ...]
+# Default models: google/gemini-2.5-flash google/gemini-3-flash-preview
+#
+# A calibration that misses the 9/10 bar exits non-zero — that's expected
+# data, not a script error. So we deliberately do NOT use `set -e` here.
+set -uo pipefail
+
+cd "$(git rev-parse --show-toplevel)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)